cmd/controller-manager/main.go

```diff
@@ -73,10 +73,10 @@ func parseFlags() {
 		"Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.")
 	flag.StringVar(&certsDir, "certs", "/etc/webhook/certs",
 		"The directory for storing certs key file and cert file")
-	flag.StringVar(&configDir, "conf", "/etc/webhook/conf",
-		"The directory for storing webhook config files")
-	flag.StringVar(&watcherConfig.Namespace, "configmap-namespace", "",
+	flag.StringVar(&watcherConfig.Namespace, "template-namespace", "",
 		"Namespace to search for ConfigMaps to load Injection Configs from (default: current namespace)")
+	flag.Var(&templateWatcherLabels, "template-labels",
```

Comment:
This is the same as the current flag `configmap-labels`. It doesn't have the default value as well. Do I need to add that too?

Reply:
OK. I just added a check function to see if the labels are nil. I think it is better to just return error if no labels are specified instead of adding a default value.